client/my-sites/scan/main.tsx

```diff
@@ -185,7 +185,7 @@ class ScanPage extends Component< Props > {
 				) }
 				<p>
 					{ translate(
-						'We will send you an email once the scan completes, in the meantime feel ' +
+						'We will send you an email if security threats are found. In the meantime feel ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'We will send you an email once the scan completes, in the meantime feel free to continue to use your site as normal, you can check back on progress at any time.' )` *ES Score: 6*